src/registry_schemas/schemas/party.json

```diff
@@ -56,11 +56,19 @@
                 "person": {
                     "$ref": "#/definitions/person"
                 },
-                "deliveryAddress": {
-                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
-                },
-                "mailingAddress": {
-                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                "address": {
+                    "type": "object",
+                    "properties": {
+                        "deliveryAddress": {
+                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                        },
+                        "mailingAddress": {
+                            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                        }
+                    },
+                  "required": [
+                      "mailingAddress"
```

Comment:
I think it's delivery address that is the minimum required address

Reply:
For incorporations it is the other way round. Mailing address is the mandatory one and delivery address is mandatory only if the party has a  Director role

Reply:
In the create businesses UI for persons/incorporators only mailing address is required/collected whereas Directors also ( potentially ) have a delivery.

Reply:
ah interesting, so I guess this would be in the legal api validator